deltachat-ios/Chat/Views/Cells/VideoInviteCell.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import UIKit
+import DcCore
+
+public class VideoInviteCell: UITableViewCell {
+
+    private lazy var messageBackgroundContainer: BackgroundContainer = {
+        let container = BackgroundContainer()
+        container.image = UIImage(color: DcColors.systemMessageBackgroundColor)
+        container.contentMode = .scaleToFill
+        container.clipsToBounds = true
+        container.translatesAutoresizingMaskIntoConstraints = false
+        return container
+    }()
+
+    lazy var avatarView: InitialsBadge = {
+        let view = InitialsBadge(size: 28)
+        view.setColor(DcColors.systemMessageBackgroundColor)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        return view
+    }()
+
+    lazy var videoIcon: InitialsBadge = {
```

Comment:
i would like to have the video icon a tiny bit smaller (circle should stay as is) and closer to the avatar icon, cmp android.

maybe that already makes the circle more visible, otherwise, we should add a bit more contrast here (is that already the same shading as used eg. for date labels and info messages?)

both are no blockers, however.

Reply:
ok, I did both, reduced the size of the video Icon and darkened a little bit the background color for system messages